### DISCUSSION TITLE
Removing Entities permanently from en_core_web_trf

### DISCUSSION THREAD

**Discussion by hansm4305** [explosion/spaCy#11660] [Help: Coding & Implementations] [answered]
Hi 

I am working on an entity detection where I want to detect only two entities PERSON and WORK_OF_ART. 

As an example

Khatia Buniatishvili - Chopin: Prelude in E minor Op. 28, no. 4
Khatia Buniatishvili & Ballet Preljocaj - Mozart: Piano Concerto no. 23/II. Adagio
Khatia Buniatishvili - Liszt: Three Concert Études S. 144, no. 2 "La leggierezza"

In these the first part is PERSON which is correctly detected and second part is WORK_OF_ART but since numbers are appearing in second part, they are detected as CARDINAL. So inorder to detect these two entities alone (PERSON and WORK_OF_ART), then 
1. is it possible to remove all other entities from the pretrained model so that other entities wont be affecting these two.
2. If possible to remove, how it can be removed

Thanks

**Comment by thomashacker** [comment, marked as answer]
Hello,
It's not possible to remove a label because the model learned all labels in combination. Also, `WORK_OF_ART` does not perform well compared to the other labels.
